### PR TITLE
Skip PagerDuty integration when there is no CephCluster available

### PR DESCRIPTION
**Pull request by fbalak**
Fixes https://github.com/red-hat-storage/ocs-ci/issues/5397

Signed-off-by: Filip Balak <fbalak@redhat.com>

### CODE REVIEW

**Comment by fbalak** [comment]
Verified: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/16086/

File: tests/conftest.py [hunk 1]
```diff
@@ -472,12 +479,19 @@ def pagerduty_integration(request, pagerduty_service):
...
+    teardown = config.RUN["cli_params"]["teardown"]
+    skip_ocs_deployment = config.ENV_DATA["skip_ocs_deployment"]
+    ceph_cluster_installed = config.RUN.get("cephcluster")
+    if teardown or skip_ocs_deployment or not ceph_cluster_installed:
+        log.info("CephCluster is not available. Skipping PagerDuty integration.")
+        return None
+    if (
+        config.ENV_DATA["platform"].lower() not in constants.MANAGED_SERVICE_PLATFORMS
+        or config.ENV_DATA.get("disable_pagerduty")
+        or (teardown or skip_ocs_deployment or not ceph_cluster_installed)
```
Comment (ebondare) [inline tests/conftest.py:491]: Haven't you already taken care of these cases by returning None on line 487?

Comment (fbalak) [reply tests/conftest.py:488]: Yes, updated

**Review by mbukatov — APPROVED** [review]

**Review by ebondare — APPROVED** [review]
LGTM

**Pull request merged**